app/app.go

```diff
@@ -428,12 +431,20 @@ func New(
 		app.BankKeeper,
 	)
 
+	app.oracleKeeper = *oraclekeeper.NewKeeper(
```

Comment:
This oracle setup is already set in another PR.
I will change this code when merging